include/lsst/jointcal/Jointcal.h

```diff
@@ -21,7 +21,7 @@ namespace jointcal {
         LSST_CONTROL_FIELD(sourceFluxField, std::string, "name of flux field in source catalog");
 
         JointcalControl() :
-            sourceFluxField("base_CircularApertureFlux_7")
+            sourceFluxField("None")
```

Comment:
I'm not convinced this is doing what you think it is doing, or that perhaps I don't understand something about LSST_CONTROL_FIELDS. You are setting it to "None" so that a user must specify a parameter. However "None" is still a string, and thus validate() will still succeed and the program will fail later when it tries to find the None in the schema. Wouldn't it be better to leave the control field blank, so that validate the control early, and have it throw a message about needing to set it? Again, maybe I am just not following what the control field is doing and "None" has a special meaning here.